RadarIO.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
 	<metadata>
 		<id>RadarIO.Xamarin</id>
-		<version>3.9.3-beta.2</version>
+		<version>3.9.3-beta.3</version>
```

Comment:
just saying shouldn't this have been 3.9.4-beta since you have a stable 3.9.3???
<img width="869" height="262" alt="Image" src="https://github.com/user-attachments/assets/3f0e1362-a513-4363-9187-a889dd71605c" />

Reply:
The 3.9.3 reflects the version of the wrapped native sdk. You’re right, though, that the ordering for 3.9.3 packages has gotten mixed up due to the prerelease suffix. This will be resolved in the forthcoming 3.21.4 release.